lib/dbservice/groups.ex

```diff
@@ -45,16 +45,16 @@ defmodule Dbservice.Groups do
   end
 
   @doc """
-  Gets group by child_id.
+  Gets group by child_id and type.
   Raises `Ecto.NoResultsError` if the Group does not exist.
```

Comment:
Update comment also to say that error will be returned if type is wrong or does not exist

Reply:
Ok, I will update the comment.